e2e/main_test.go

```diff
@@ -107,33 +105,35 @@ func mockEnvoyVersionsServer() (*httptest.Server, error) {
 	return ts, nil
 }
 
-// readFuncEPath reads E2E_FUNC_E_BINARY or defaults to "$PWD/dist/func-e_$GOOS_$GOARCH/func-e"
+// readFuncEBin reads E2E_FUNC_E_PATH or defaults to "../func-e"
 // An error is returned if the value isn't an executable file.
-func readFuncEPath() (string, error) {
-	path := os.Getenv(funcEBinaryEnvKey)
-	if path == "" {
-		// Assemble the default created by "make bin"
-		relativePath := filepath.Join("..", "dist", fmt.Sprintf("func-e_%s_%s", runtime.GOOS, runtime.GOARCH), "func-e")
-		abs, err := filepath.Abs(relativePath)
-		if err != nil {
-			return "", fmt.Errorf("%s didn't resolve to a valid path. Correct environment variable %s", path, funcEBinaryEnvKey)
+func readFuncEBin() error {
+	path := os.Getenv(funcEPathEnvKey)
+	path = filepath.Join("..", path) // relative to project root
```

Comment:
looks like path defaults to `../${E2E_FUNC_E_PATH}`?

Reply:
what happens is that pwd is /path/to/project/e2e, so the ../ rebases it at the project root.

ex E2E_FUNC_E_PATH=dist/whatever
/path/to/project/e2e/../dist/whatever = /path/to/project/dist

Reply:
so we cannot pass the full path to `E2E_FUNC_E_PATH`?

Reply:
I mean, absolute path

Reply:
hmm maybe this would fail if an absolute path were given instead of a relative one. Ex /usr/local/bin. I'll rework it